projects/schematics/src/migrations/4_0/removed-public-api-deprecations/removed-public-api-deprecation.ts

```diff
@@ -34,6 +35,36 @@ export const REMOVED_PUBLIC_API_DATA: DeprecatedNode[] = [
   //   importPath: SPARTACUS_STOREFRONTLIB,
   //   comment: `'${VARIANT_STYLE_SELECTOR_COMPONENT}' was moved to @spartacus/product/variants.`,
   // },
+  // projects/core/src/personalization/personalization.module.ts
+  {
+    node: 'PersonalizationModule',
```

Comment:
I think we should make make these filename available in `projects/schematics/src/shared/constants.ts` for future usage and to stay consistent on how we do it too